.gitignore

```diff
@@ -1,3 +1,4 @@
 stacks-node/persistent-data
 node_modules
 data.json
+.env
```

Comment:
AS we are removing the .env let's create a .env.example so users know what to add in local

Reply:
@pradel added